unlock-app/src/components/lock/Overlay.js

```diff
@@ -95,17 +96,43 @@ const Locks = styled.ul`
   list-style: none;
   margin: 0px;
   padding: 0px;
+  grid-row: 2;
+  grid-column: 1;
 `
 
 const Colophon = styled.footer`
-  display: grid;
-  justify-content: center;
+  display: flex;
+  flex-direction: row;
+  align-content: center;
   font-family: 'Roboto', sans-serif;
   font-weight: 300;
   font-size: 12px;
   color: var(--darkgrey);
+  background-color: var(--white);
+  justify-self: right;
+  align-self: center;
+  grid-row: 2;
```

Comment:
Since you changed to `display: flex;` does this have any effect? Same with `grid-column` below? I thought (but could be wrong) that these are `display: grid;` features?

Reply:
yes they are. The `<footer>` element is located in its parent's grid, and the `grid-row` and `grid-column` values position it in its parent's grid. I think the naming is rather confusing, since almost every other thing that refers to self has "self" in the title, but there you have it.  Without these, the element displays wherever the hell it wants (usually in page order)